devel/ansible/roles/datagrepper/tasks/main.yml

```diff
@@ -0,0 +1,120 @@
+---
+- name: Install RPM packages
+  dnf:
+      name:
+      - fedora-messaging
+      - gcc
+      - git
+      - poetry
+      - python3-devel
+      - python3-pip
+      - vim
+      - krb5-devel
+      - libpq-devel
+      - tox
+      - python-psycopg2
+      state: present
+
+- name: Install the .bashrc
+  copy:
+    src: .bashrc
+    dest: /home/vagrant/.bashrc
+    mode: 0644
+    owner: vagrant
+    group: vagrant
+
+- name: check out datanommer from git
+  git:
+    repo: https://github.com/fedora-infra/datanommer.git
+    dest: /home/vagrant/datanommer
+  become: yes
+  become_user: vagrant
+
+- name: Create a directory for the virtualenv
+  file:
+    name: /srv/venv
+    state: directory
+    mode: 0755
+    owner: vagrant
+    group: vagrant
+
+- name: Create the virtualenv and install poetry
+  pip:
+    name: poetry
+    virtualenv: /srv/venv
+  become: yes
+  become_user: vagrant
+
+- name: Install datagrepper with poetry
+  shell: /srv/venv/bin/poetry install
+  args:
+    chdir: /home/vagrant/datagrepper/
+  become: yes
+  become_user: vagrant
+
+- name: Install datanommer.models with poetry
+  shell: /srv/venv/bin/poetry install
+  args:
+    chdir: /home/vagrant/datanommer/datanommer.models/
```

Comment:
That's apparently still not enough, because datagrepper installs a version of `datanommer.models` that takes precedence over the development install that poetry does.

On my fresh Vagrant machine, running the consumer fails on the first message because it's calling the old version of `datanommer.models.add()` (which does not know how to handle FM messages). And when I look at `/srv/venv/lib/python3.9/site-packages/datanommer/models/__init__.py`, it's the old version. Maybe running `pip uninstall datanommer.models` before running `poetry install` would help ?

Reply:
Another sign of the same issue: the database created with `datanommer-create-db` is in the old format.